test/stat_tracker_test.rb

```diff
@@ -28,15 +28,27 @@ def test_it_exists
   #   assert_equal './data/game_teams_fixture.csv', @stat_tracker.game_teams_path
   # end
 
-  # Game Statistics Tests - Helper Methods #
+# Game Statistics Tests - Helper Methods #
   def test_it_can_get_total_goals_per_game
     assert_equal 5, @stat_tracker.total_goals_per_game[2012030221]
     assert_equal 3, @stat_tracker.total_goals_per_game[2012030231]
   end
 
-  # Game Statistics Tests - Stat Methods #
+  def test_it_can_get_away_wins
+    assert_equal 13, @stat_tracker.total_away_wins
+  end
+
+# Game Statistics Tests - Stat Methods #
   def test_it_can_get_highest_total_score
-    assert_equal 7, @stat_tracker.highest_total_score
+    assert 5, @stat_tracker.highest_total_score
```

Comment:
This should be an assert_equal but I think Leah already corrected it in one of her PRs

Reply:
Another merge error. I thought there was something odd about that one.